install.sh

```diff
@@ -61,10 +61,10 @@ sudo pip3 install python-dotenv
 
 sudo apt-get install git
 
-git clone -b master https://github.com/OriginTrail/ot-node.git
+git clone -b develop https://github.com/OriginTrail/ot-node.git
```

Comment:
@vukdjoric 
Out of curiosity, why do we want to pull latest develop instead of master?